### DISCUSSION TITLE
HomeAssistant Addon und HB-RF-ETH

### DISCUSSION THREAD

**Discussion by Fulleranio** [OpenCCU/OpenCCU#2547] [🇩🇪 Fragen&Antworten] [answered]
Hallo there,
I'am new here an have been working with homeassistant and raspberrymatic for quite a while now. Lately I wanted to to change my platform to homeassistant yellow. My idea was to use the HB-RF-ETH since it offers a remote antenna which I need because our house is quite long. But I cannot find the /ect/config file in the homeassistant-raspberrymatic installation. Perhaps someone already knows how to set the hole set up. I would be glad if someone ansers

**Comment by Baxxy13** [comment, marked as answer]
Zugriff auf die Datei ist nicht notwendig.
In der Systemsteuerung --> erweiterte Einstellungen kann die ETH konfiguriert werden. Danach muss das RaspberryMatic - AddOn neu gestartet werden.

**Reply by Fulleranio** [reply]
Vielen Dank. Sie funktioniert jetzt.
